### ISSUE TITLE
Example app use Android embed v1

### ISSUE THREAD

**Issue by enricobenedos** [OneSignal/OneSignal-Flutter-SDK#439]
Simply clone this repository and start the example on iOS simulator:
```console
━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
Warning
──────────────────────────────────────────────────────────────────────────────
Your Flutter application is created using an older version of the Android
embedding. It's being deprecated in favor of Android embedding v2. Follow the
steps at
https://flutter.dev/go/android-project-migration
to migrate your project.
━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
```
```console
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, 2.2.3, on macOS 11.4 20F71 darwin-x64, locale en-IT)
[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.2)
[✓] Xcode - develop for iOS and macOS
[✓] Chrome - develop for the web
[✓] Android Studio (version 4.2)
[✓] VS Code (version 1.58.2)
[✓] Connected device (4 available)

• No issues found!
```

**Comment by jkasten2** [comment]
@enricobenedos Thanks for reporting, I think you meant Android emulator instead of iOS 😄 . 
The SDK has been updated to the Android embed v2 API in PR #486 and was included in the [3.2.4 release](https://github.com/OneSignal/OneSignal-Flutter-SDK/releases/tag/3.2.4).

**Comment by enricobenedos** [comment]
Hello @jkasten2, yes I mean Android Emulator!
